examples/serviceconfig.json

```diff
@@ -1,15 +1,16 @@
 {
-  "disable_persistent_config": true,
-  "adminInterface": "127.0.0.1:4985",
-  "interface": "0.0.0.0:4984",
-  "databases": {
-    "db": {
-      "server": "walrus:data",
-      "users": {
-        "GUEST": {"disabled": false, "admin_channels": ["*"] }
-      },
-      "allow_conflicts": false,
-      "revs_limit": 20
+  "bootstrap": {
+    "server": "rosmar:/?mode=memory"
+  },
+  "api": {
+    "public_interface": ":4984",
+    "admin_interface": "127.0.0.1:4985"
+  },
+  "logging": {
+    "console": {
+      "enabled": true,
+      "log_level": "info",
+      "log_keys": ["*"]
     }
```

Comment:
I think I'd be inclined to not have console logging at info level, this is just going to fill up systemd logs and the log files are still written to the log file directory.

I use console logging like this when I run SGW myself because it's nice to have it write to stdout but I don't think that's how it is used when it is controlled by systemd